mkninja.sh

```diff
@@ -541,6 +542,7 @@ encodedecodetest ibm180_525
 encodedecodetest ibm360_525
 encodedecodetest ibm720
 encodedecodetest ibm720_525
+encodedecodetest ibm_n88basic
```

Comment:
I think just calling this `n88basic` will work, as this has nothing to do with standard IBM formats. This was on the PC-8800, right? Was it used by anything other than BASIC? If so, calling it `pc8800` might be better.

Reply:
I can do that too, it's still a standard IBM FDC with normal MFM/FM on it which is why I left the IBM prefix.

It's used by games too but mostly those written in BASIC. It's also used by the PC-98's implementation of N88-Basic. The PC-88 supports pretty arbitrary sector layouts, so I don't think there is any one format I'd call pc8800.

Reply:
We have other normal IBM formats without the prefix --- Acorn ADFS and DFS, for example. The `ibm` ones are just the ones generic enough they're used by more than one platform. This one's hardware specific so I think `n88basic` is most appropriate.

Reply:
It's been updated to remove the prefix.